### DISCUSSION TITLE
Uh...this is incredible.

### DISCUSSION THREAD

**Discussion by kerns** [dromru/react-photoswipe-gallery#1011] [General]
Photoswipe is hands down the best gallery ever written. The latest version and ES6 re-write only cement that legacy further. The time you've invested bringing this goodness to React in a coherent way,...the *exhaustive* documentation with examples... it's just an amazing resource.

Please just keep it maintained and continue the great work. 🙏 It is _really_ appreciated.

**Comment by irudoy** [comment]
Great to hear, thank you! This is encouraging and inspiring.
There is no doubt that we will continue working on it.
